$OpenBSD: patch-cmake_modules_KDevPlatformMacros_cmake,v 1.2 2015/07/10 18:28:14 zhuk Exp $
Can't use "$" matching here because (G)Make interprets it sooner or later.
--- cmake/modules/KDevPlatformMacros.cmake.orig	Mon Feb  2 17:14:28 2015
+++ cmake/modules/KDevPlatformMacros.cmake	Fri Jul  3 01:21:31 2015
@@ -52,7 +52,8 @@ macro(kdevplatform_create_template_archive _templateNa
     else(WIN32)
         add_custom_command(OUTPUT ${_template}
             COMMAND tar ARGS -c -C ${CMAKE_CURRENT_SOURCE_DIR}/${_templateName}
-                --exclude .kdev_ignore --exclude .svn
+                -s "@^\\(.*/\\)*\\.kdev_ignore@@"
+                -s "@^\\(.*/\\)*\\.svn@@"
                 -j -f ${_template} .
             DEPENDS ${_deps}
         )
